### ISSUE TITLE
ArduPilot 4.6 issues list

### ISSUE THREAD

**Issue by rmackay9** [ArduPilot/ardupilot#28612]
This is a list of issues discovered during ArduPilot 4.6 testing.  Issues for [Copter/Rover-4.5 issues are here](https://github.com/ArduPilot/ardupilot/issues/26103)

**Reports requiring investigation**

- [ ] https://github.com/ArduPilot/ardupilot/issues/32344
- [ ] https://github.com/ArduPilot/ardupilot/issues/31484
- [ ] Internal Error (flow-of-control) during takeoff in AltHold ([discussion](https://discuss.ardupilot.org/t/copter-4-6-3-released/140596/4))
- [ ] F.Port not working with TradHeli ([discussion](https://discuss.ardupilot.org/t/trad-heli-tuning-fport-can-t-send-telemetry-back/138940/17))
- [ ] https://github.com/ArduPilot/ardupilot/issues/31287
- [ ] https://github.com/ArduPilot/ardupilot/issues/31099
- [ ] EKF reset leads to bad roll estimate ([issue](https://github.com/ArduPilot/ardupilot/issues/31098), [discussion](https://github.com/ArduPilot/ardupilot/issues/31098))
- [ ] BLHeli_S ESC passthrough not working? ([discussion](https://discuss.ardupilot.org/t/copter-4-6-2-released/136822/26))
- [ ] yaapu ver X7 1.8.0 using Taranis QX7 receiver not displaying data ([discussion](https://discuss.ardupilot.org/t/copter-4-6-2-released/136822/12))
- [ ] log download slow? ([discussion](https://discuss.ardupilot.org/t/copter-4-6-1-released/135697))
- [ ] Plane LEDs not working? ([issue](https://github.com/ArduPilot/ardupilot/issues/30331#issuecomment-2988401515), [PR with potential fix](https://github.com/ArduPilot/ardupilot/pull/30469))
- [ ] Bad battery ([discussion](https://discuss.ardupilot.org/t/copter-4-6-0-released/134617/74))
- [ ] Micoair h743aio failing to restart ([discussion](https://discuss.ardupilot.org/t/copter-4-6-0-released/134617/57))
- [ ] Bad Gyro Health on Yaapu ([discussion](https://discuss.ardupilot.org/t/copter-4-6-0-released/134617/50))
- [ ] https://github.com/ArduPilot/ardupilot/issues/30142
- [ ] Rover follow Lua bindings almost always returning nil ([discussion](https://discuss.ardupilot.org/t/issue-with-follow-object-in-lua-script-for-rover-to-follow-copter/131666/11))
- [ ] https://github.com/ArduPilot/ardupilot/issues/30173 ([discussion](https://discuss.ardupilot.org/t/cube-orange-plus-over-loaded-processor/134003/41))
- [ ] https://github.com/ArduPilot/ardupilot/issues/30156
- [ ] Solo gimbal not included on CubeSolo ([discussion](https://discuss.ardupilot.org/t/copter-4-6-0-released/134617/5))
- [ ] uBlox fw update over DroneCAN with uCenter 24.x does not work
- [ ] Since 4.6 NeoPixel strip led doesn’t work https://discuss.ardupilot.org/t/plane-4-6-release/134619/15?u=tridge
- [ ] garbage data can be accepted as valid mavlink: https://github.com/ArduPilot/pymavlink/issues/1019
- [ ] FTP mission upload deadlock in 4.6.2: AP: CRITICAL Deadlock 814 FTP 0x3000AE6C
- [ ] T-Motor DroneCAN ESCs get very low ESC status rate and thus RPM rate
- [ ] https://github.com/ArduPilot/ardupilot/issues/32087

**Confirmed issues**

- [ ] https://github.com/ArduPilot/ardupilot/issues/31584
- [ ] https://github.com/ArduPilot/ardupilot/issues/31968
- [ ] RC tuning of Loiter speed not working? ([discussion](https://discuss.ardupilot.org/t/copter-4-6-3-released/140596/41))
- [ ] AHRS orientation on startup ([discussion](https://discuss.ardupilot.org/t/copter-4-6-3-released/140596/5), [issue](https://github.com/ArduPilot/ardupilot/issues/30064)) -- known issue
- [ ] AHRS upside soon after startup (reported by partners, [PR](https://github.com/ArduPilot/ardupilot/pull/29667), [issue](https://github.com/ArduPilot/ardupilot/issues/30064)) -- resolved for -beta6 but then re-opened
- [ ] Hardfault on arming if CAN_SLCAN_CPORT set non-zero and have SERIAL6_PROTOCOL set to 22 (reported by @robertlong13, [Issue](https://github.com/ArduPilot/ardupilot/issues/30055)) -- also exists in 4.5 so not a blocker
- [x] Internal error involved multiheap allocation for LUA in cygwin (https://github.com/ArduPilot/ardupilot/issues/29563)
- [ ] EKF3 altitude runaway EK3_SRC1_POSZ = 2 (rangefinder) and rangefinder is out of range ([PR](https://github.com/ArduPilot/ardupilot/pull/29537))
- [ ] AM32 passthrough (without DroneCAN) not working ([issue](https://github.com/ArduPilot/ardupilot/issues/29348), [PR](https://github.com/ArduPilot/ardupilot/pull/29590))
- [ ] EKF3 velocity divergence before arming ([issue](https://github.com/ArduPilot/ardupilot/issues/29384), [PR](https://github.com/ArduPilot/ardupilot/pull/29434)) -- hoping PaulR will investigate (not a blocker)
- [x] https://github.com/ArduPilot/ardupilot/issues/31108
- [ ] https://github.com/ArduPilot/terraingen/pull/23

***Plane Specific Issues***

- [ ] Plane servos using IOMCU not moving ([issue](https://github.com/ArduPilot/ardupilot/issues/29759)) -- aiming for fix in 4.6.x point release (not 4.6.0)
- [ ] the assisted_flight flag oscillates rapidly in fwd transition of tilt quadplanes, possibly leading to incorrect tllt slew rate https://github.com/ArduPilot/ardupilot/issues/28957 -- aiming to fix in 4.6.1 (or higher)
- [ ] possible landing glide slope bug https://discuss.ardupilot.org/t/plane-4-6-release/134619/8?u=tridge
- [ ] possible sink rate issue in AUTOLAND https://discuss.ardupilot.org/t/autoland-mode-4-7-dev-reports/135441
- [ ] https://github.com/ardupilot/ardupilot/issues/30227

**Enhancements or Issues that may not be resolved before the stable release**

- [ ] Octaquad missing on F4 boards ([discussion](https://discuss.ardupilot.org/t/bug-in-firmware-upgrade-copter-4-6-0/134683/30)) -- same with 4.5.7 so we should potentially also remove Octa from F4 boards
- [ ] BLHeli SERVO_BLH_RVMASK mapping is achieved through trial and error (raised by xfacta, [issue](https://github.com/ArduPilot/ardupilot/issues/29325))
- [ ] when the user has configured ESC, FFT or RPM notch and has INS_HNTCx_REF=0 then they end up with a fixed notch. We should have an arming check to tell them they need to set the REF to 1.0 (any non-zero value will do actually) https://github.com/ArduPilot/ardupilot/issues/28786
- [ ] ROS2 attitude control + thrust enhancement request ([discussion](https://discuss.ardupilot.org/t/ros2-interface-for-controlling-thrust-and-attitude/127694))
- [ ] AutoTune should have a vehicle specific (or configurable) AngleP max (reported by Tridge)
- [ ] add a warning if the GCS system clock and UTC clock on the flight controller differs by over 20s and signing is active, as signing will fail with a 30s difference
- [ ] Custom Build Server for CubeOrangePlus with OpenDroneID shows linker error when out of flash ([discussion1](https://discuss.ardupilot.org/t/copter-4-6-0-beta1-is-available-for-beta-testing/126295/4), [discussion2](https://discuss.ardupilot.org/t/copter-custom-firmware-build-issue-with-remote-id-support-latest-not-4-5-7/125177), [Issue](https://github.com/ArduPilot/CustomBuild/issues/114))
- [ ] Add LDRobot LD19 support ([discussion](https://discuss.ardupilot.org/t/copter-4-6-0-beta1-is-available-for-beta-testing/126295/8), [issue](https://github.com/ArduPilot/ardupilot/issues/28662))
- [ ] need to provide a clearer message to users when the power status flags change, indicating loss of a power source in flight. A statustext at message level CRITICAL I think is warranted, maybe "POWER STATUS CHANGE" ? (Tridge has details)

**Wiki issues/enhancement requests ([Wiki To-Do list](https://github.com/ArduPilot/ardupilot_wiki/issues/4274))**

- None

**Resolved issues**

- [x] https://github.com/ArduPilot/ardupilot/issues/31953 -- fixed in 4.7, could be backported to 4.6.4
- [s] https://github.com/ArduPilot/MissionPlanner/issues/3661 -- misunderstanding
- [x] https://github.com/ArduPilot/ardupilot/issues/31402 -- resolved for 4.7
- [x] https://github.com/ArduPilot/ardupilot/issues/31386 -- resolved for 4.7
- [x] Rover: "field elevation set 0m" spamming GCS messages ([discussion](https://discuss.ardupilot.org/t/how-to-disable-a-continuous-field-elevation-set-0m-message-sent-on-console-every-sec-sitl-ardurover-4-5-5/122567), [PR](https://github.com/ArduPilot/ardupilot/pull/25114))
- [x] Ch6 tuning of WPSpeed not working on tradheli ([discussion1](https://discuss.ardupilot.org/t/copter-4-6-3-released/140596/31), [discussion2](https://discuss.ardupilot.org/t/wpnav-speed-as-a-chan-6-tune-parameter-working/83217/18)) -- user error
- [x] Lua related Internal error from multi-heap allocation ([issue](https://github.com/ArduPilot/ardupilot/issues/29563))
- [x] https://github.com/ArduPilot/ardupilot/issues/31404 -- resolved through documentation
- [x] incorrect glide slope for terrain NAV_LAND. https://github.com/ArduPilot/ardupilot/pull/30041
- [x] find_by_name() in AP_Param.cpp uses a bogus strncasecmp() ([PR](https://github.com/ArduPilot/ardupilot/pull/30659))
- [x] https://github.com/ArduPilot/ardupilot/issues/30722 -- resolved ahead of 4.6.3-beta1
- [x] Firmware erase on ECC error on boards that store params in flash ([PR](https://github.com/ArduPilot/ardupilot/pull/30031)) -- TBS bootloader released with 4.6.0, what's next? -- solved in 4.6.2
- [x] LD06 sensitive to sunlight ([discussion](https://discuss.ardupilot.org/t/copter-4-6-1-released/135697/31)) -- not a regression since 4.5.  hardware issue?
- [x] Ark FPV board mentioned in release notes but not in 4.6 ([discussion](https://discuss.ardupilot.org/t/copter-4-6-0-released/134617/83), [PR](https://github.com/ArduPilot/ardupilot/pull/28603)) -- marked for backporting, may be included in 4.6.3
- [x] JFB110 SD card not working? ([discussion](https://discuss.ardupilot.org/t/jfb110-fc-mavftp-issue/134938), [related PR](https://github.com/ArduPilot/ardupilot/pull/26090), [potential fix PR](https://github.com/ArduPilot/ardupilot/pull/30387)) -- resolved for 4.6.2-beta1
- [x] Copter Pixhawk 2.4.8 "check frame and type" ([discussion](https://discuss.ardupilot.org/t/bug-in-firmware-upgrade-copter-4-6-0/134683)) -- no response from user
- [x] CubeOrange AK09916 internal compass orientation Yaw270 ([discussion](https://discuss.ardupilot.org/t/copter-4-6-0-released/134617/44)) -- no functional impact
- [x] “pitch level-off at takeoff altitude” is interfered with by the feature “pitch rotate speed for ground takeoff” ([discussion](https://discuss.ardupilot.org/t/plane-4-6-0-beta/126358/165), [PR](https://github.com/ArduPilot/ardupilot/pull/29834)) -- resolved for 4.6.0 (randy thinks)
- [x] DroneCan M10 GPS lock issue again (raised by TimT, [discussion](https://discuss.ardupilot.org/t/timts-dronecan-ublox-m10-num-sats-issue/132353)) -- waiting on Qiotek, no conclusive evidence of an issue
- [x] LD19 average change makes it sensitive to sunlight ([discussion](https://discuss.ardupilot.org/t/looking-to-get-firmware-release-4-6-0-beta2-11-dec-2024/133422), [original PR](https://github.com/ArduPilot/ardupilot/pull/27045), [PR1](https://github.com/ArduPilot/ardupilot/pull/30016), [PR2](https://github.com/ArduPilot/ardupilot/pull/30107)) -- resolved ahead of 4.6.1-beta1
- [x] CubeOrangePlus 2nd IMU error ([discussion](https://discuss.ardupilot.org/t/imu-2-on-my-cubeorangeplus-errors/133691)) -- need log, no feedback from user
- [x] SpeedyBee F4 running out of memory ([discussion](https://discuss.ardupilot.org/t/possible-side-problem/133917/2)) -- need log, no feedback from user
- [x] Pixhawk6X IST8310 external compass not detected ([issue](https://github.com/ArduPilot/ardupilot/issues/29854)) -- need log, no feedback from user
- [x] TBS parameter corruption on firmware update (reported by AndyP, [Issue](https://github.com/ArduPilot/ardupilot/issues/30058), [PR](https://github.com/ArduPilot/ardupilot/pull/30031)) -- TBS bootloaders released with 4.6.0
- [x] AIS not working on Copter? ([discussion](https://discuss.ardupilot.org/t/ais-no-work-daisy2-ac-45-46/131176), [related PR](https://github.com/ArduPilot/ardupilot/pull/29839#event-17337598051)) -- cannot reproduce, no response from user
- [x] Secure bootloader doesn't work ([4.5 discussion](https://discuss.ardupilot.org/t/unable-to-update-signed-bootloader/133280/2)) -- could not reproduce
- [x] Speedybee ENOMEM message ([discussion](https://discuss.ardupilot.org/t/logging-failed-failed-to-create-log-directory-apm-logs-enomem/132917), [wiki PR](https://github.com/ArduPilot/ardupilot_wiki/pull/6768)) -- resolved, wiki updated ahead of -beta6
- [x] "No ap_message for mavlink id (271)" message ([discussion](https://discuss.ardupilot.org/t/copter-4-6-0-beta5-available-for-beta-testing/132624/23), [PR](https://github.com/ArduPilot/ardupilot/pull/29783)) -- resolved for -beta6
- [x] terrain following issue: https://github.com/ArduPilot/ardupilot/pull/29685 -- resolved for beta6
- [x] circle climb after terrain: https://github.com/ArduPilot/ardupilot/pull/29765 -- resolved for beta6
- [x] Microbrain UAV-R21 Proximity sensor ([discussion](https://discuss.ardupilot.org/t/copter-4-6-0-beta4-released-for-beta-testing/130471/48), [discussion2](https://discuss.ardupilot.org/t/how-to-use-the-uav-h31-and-uav-r21-for-terrain-following-and-obstacle-avoidance/114423)) -- we do not support this model
- [x] RPI PiPilot / Invensense ICM42688P hang ([issue](https://github.com/ArduPilot/ardupilot/issues/29572), [PR](https://github.com/ArduPilot/ardupilot/pull/29614)) -- resolved for beta6
- [x] on ICM45686 IMUs the SPI transfer overheads can be overwhelming on some boards, especially those with 3 of that sensor. Some PRs to address: https://github.com/ArduPilot/ardupilot/pull/28672 and https://github.com/ArduPilot/ardupilot/pull/27573 -- resolved for -beta6
- [x] [IMU temp cal pages](https://ardupilot.org/copter/docs/common-imutempcal.html) need updating -- no details on what needs updating
- [x] Copter/TradHeli internal errors 0X4000000 and 0X4000020 ([discussion](https://discuss.ardupilot.org/t/copter-4-6-0-beta4-released-for-beta-testing/130471/23), [issue](https://github.com/ArduPilot/ardupilot/issues/29370), [PR](https://github.com/ArduPilot/ardupilot/pull/29409)) -- resolved for -beta5
- [x] SET_ROI_LOCATION support ([issue](https://github.com/ArduPilot/ardupilot/issues/28505#issuecomment-2709173400), [PR](https://github.com/ArduPilot/ardupilot/pull/29567)) -- resolved ahead of -beta5
- [x] Take Pic with no location ([issue](https://github.com/ArduPilot/ardupilot/issues/29530), [PR](https://github.com/ArduPilot/ardupilot/pull/29558)) -- fixed in master (won't backport)
- [x] Pixhawk 5x ethernet not enabled ([issue](https://github.com/ArduPilot/ardupilot/issues/26397#issuecomment-2725220803), [wiki PR](https://github.com/ArduPilot/ardupilot_wiki/pull/6697)) -- wiki PR merged ahead of -beta5
- [x] AP_Scripting should get follow binding for Rover ([discussion](https://discuss.ardupilot.org/t/issue-with-follow-object-in-lua-script-for-rover-to-follow-copter/131666/4), [PR](https://github.com/ArduPilot/ardupilot/pull/29548)) -- resolved for -beta5
- [x] Plane: fixed guided VTOL command in terrain mode ([PR](https://github.com/ArduPilot/ardupilot/pull/28406))
- [x] ARSPD_AUTOCAL can trigger while disarmed ([PR](https://github.com/ArduPilot/ardupilot/pull/29455)) -- resolved for -beta5
- [x] QuadPlane motors using BDShot making knocking noises ([discussion1](https://discuss.ardupilot.org/t/plane-4-6-0-beta/126358/62), [discussion2](https://discuss.ardupilot.org/t/plane-4-6-0-beta/126358/52), [PR1](https://github.com/ArduPilot/ardupilot/pull/29518), [PR2](https://github.com/ArduPilot/ardupilot/pull/29521)) -- fixed for beta5
- [x] Water Depth message too fast on boats ([discussion](https://discuss.ardupilot.org/t/water-depth-clogging-the-mavlink-transmission/131509), [PR](https://github.com/ArduPilot/ardupilot/pull/28877)) -- resolved for beta5
- [x] UBlox M10 can't get enough sats ([discussion](https://discuss.ardupilot.org/t/fc-tries-to-config-gps-constantly-when-trying-to-arm-in-loiter-mode/129930/15)) -- testing is inconclusive
- [x] Copter MOT_THST_HOVER param desc for lower limit should be 0.125 ([discussion](https://discuss.ardupilot.org/t/copter-4-6-0-beta4-released-for-beta-testing/130471/54), [PR](https://github.com/ArduPilot/ardupilot/pull/29478)) -- resolved for -beta5
- [x] iBUS telemetry not working? ([discussion](https://discuss.ardupilot.org/t/copter-4-6-0-beta2-is-available-for-beta-testing/127445/11)) -- user error
- [x] Mission option bit issue ([discussion](https://discuss.ardupilot.org/t/plane-4-6-0-beta/126358/42), [PR](https://github.com/ArduPilot/ardupilot/pull/29435)) -- resolved for -beta5
- [x] CADDX GM2 update slow ([discussion](https://discuss.ardupilot.org/t/copter-4-6-0-beta4-released-for-beta-testing/130471/38), [PR](https://github.com/ArduPilot/ardupilot/pull/29428)) -- resolved for -beta5
- [x] Unknown message 45 ([discussion](https://discuss.ardupilot.org/t/copter-4-6-0-beta4-released-for-beta-testing/130471/43), [discussion2](https://discuss.ardupilot.org/t/sending-unknown-message-44-on-mission-planner-with-speedybee-f405-v3-guided-mode/130303), [PR](https://github.com/ArduPilot/ardupilot/pull/29339)) -- resolved for -beta5
- [x] Holybro M10 waiting to configure GPS1 ([discussion](https://discuss.ardupilot.org/t/fc-tries-to-config-gps-constantly-when-trying-to-arm-in-loiter-mode/129930/7), [PR](https://github.com/ArduPilot/ardupilot/pull/29320)) -- resolved for -beta5
- [x] LD06 support not working? ([discussion](https://discuss.ardupilot.org/t/copter-4-6-0-beta4-released-for-beta-testing/130471/27), [PR](https://github.com/ArduPilot/ardupilot/pull/29375)) -- fixed ahead of -beta5
- [x] Reboot loop after enabling Scripting ([discussion](https://discuss.ardupilot.org/t/reboot-loop-after-enabling-scr-enable-lua-scripts/130420), [issue](https://github.com/ArduPilot/ardupilot/issues/29315)) -- user using incorrect compiler
- [x] possible regression in attitude control in quadplanes? https://github.com/ArduPilot/ardupilot/issues/29060 -- no evidence that there's been a regression despite much effort spent searching
- [x] Takeoff throttle behavior for planes without airspeed sensor changed ([issue](https://github.com/ArduPilot/ardupilot/issues/28775), [PR](https://github.com/ArduPilot/ardupilot/pull/28776)) -- intentional change
- [x] ESC Telemetry logging issue ([issue](https://github.com/ArduPilot/ardupilot/issues/28807), [PR](https://github.com/ArduPilot/ardupilot/pull/29001)) -- resolved for beta3
- [x] Community dev struggles with build options ([discussion](https://discuss.ardupilot.org/t/build-from-sources-with-custom-features/127879)) -- no response from user
- [x] https://github.com/ArduPilot/ardupilot/issues/28969
- [x] Autotune acceleration limiting is not effective ([PR](https://github.com/ArduPilot/ardupilot/pull/28911)) -- resolved for beta3
- [x] Orqa 3030 autopilot config issue ([discussion](https://discuss.ardupilot.org/t/orqa-3030-f450-issue/127483), [PR](https://github.com/ArduPilot/ardupilot/pull/28855)) -- resolved for beta3
- [x] BeagleBoneBlue RC not working with Copter ([discussion](https://discuss.ardupilot.org/t/rc-sbus-works-with-plane-but-not-copter-beaglebone-blue/127540/2)) -- user error related to running as a service
- [x] Plane takeoff not completing under TKOFF_OPTIONS[0]=0 #28685 ([PR](https://github.com/ArduPilot/ardupilot/pull/28792)) -- resolved for beta3
- [x] plane takeoff regression since 4.5.x: https://github.com/ArduPilot/ardupilot/issues/28681 -- ready for beta2
- [x] on plane when FENCE_AUTOENABLE=3 and FENCE_ENABLE=0 the fence does not enable when arming if there is a RCn_OPTION set to FENCE. This is a regression from 4.5.x. (Tridge has details) Fix: https://github.com/ArduPilot/ardupilot/pull/28729 -- ready for beta2
- [x] on plane the pre-arm check for being inside polygon fence was lost since 4.5.x. Fix: https://github.com/ArduPilot/ardupilot/pull/28729 -- ready for beta2
- [x] Navio2 GPS not working ([discussion](https://discuss.ardupilot.org/t/rover-4-6-0-beta1-is-available-for-beta-testing/126296/4), [conflicting report that it does work](https://discuss.ardupilot.org/t/copter-4-6-0-beta1-is-available-for-beta-testing/126295/9)) -- fixed by refreshing/resetting firmware
- [x] arming at mid-stick question ([discuss](https://discuss.ardupilot.org/t/arming-with-throttle-at-mid-stick/127203)) -- no issue
- [x] Crossfire receiver does not reconnect after failsafe ([issue](https://github.com/ArduPilot/ardupilot/issues/28797), [PR](https://github.com/ArduPilot/ardupilot/pull/28805)) -- resolved for -beta2
- [x] Copter AutoTune can produce very low maximum acceleration values ([issue](https://github.com/ArduPilot/ardupilot/issues/28799), [PR](https://github.com/ArduPilot/ardupilot/pull/28800)) -- resolved for -beta2
- [x] memory fragmentation and multiple memory regions mean lua scripting can fail with out of memory error even if you have plenty of free memory. Addressed in https://github.com/ArduPilot/ardupilot/pull/26517
- [x] system time may be unstable when using only first GPS timestamp, consider https://github.com/ArduPilot/ardupilot/pull/24022 or similar fix (Tridge has details)
- [x] Plane fences ignored ([issue](https://github.com/ArduPilot/ardupilot/issues/28666)) -- cannot reproduce
- [x] Plane: auto-deploy landing gear on fixed wing ([PR](https://github.com/ArduPilot/ardupilot/pull/29321))
- [x] guided error with terrain target https://github.com/ArduPilot/ardupilot/pull/29602
- [x] QuadPlane fast attitude recovery issue ([PR](https://github.com/ArduPilot/ardupilot/pull/29498))
- [x] https://github.com/ArduPilot/ardupilot/issues/29555
- [x] Plane FBWA doesn't support vectored thrust? ([issue](https://github.com/ArduPilot/ardupilot/issues/30159#issuecomment-2908030994))

**Comment by amilcarlucas** [comment]
Why not copy the open issues from 4.5 into this page to make sure they are not forgotten?

**Comment by rmackay9** [comment]
@amilcarlucas,

We could but I worry that the result will be that we have two really long lists of issues that we don't actually clear up.  If we can get focus from the dev team on investigating the issues and resolving them then that could work.  We should discuss at the next dev call I think

**Comment by menschel** [comment]
@rmackay9 
In case you want to list it separately for plane.
Issue [Takeoff not completing under TKOFF_OPTIONS[0]=0 #28685 ](https://github.com/ArduPilot/ardupilot/issues/28685)
PR attempting to fix it [Plane: Fix inability to climb the last few meters of takeoff](https://github.com/ArduPilot/ardupilot/pull/28707)

**Comment by rmackay9** [comment]
@menschel thanks for that, added!

**Comment by menschel** [comment]
I have created another issue.

[Plane 4.6 beta - Takeoff throttle behavior for planes without airspeed sensor changed from 4.5.7](https://github.com/ArduPilot/ardupilot/issues/28775)

and the corresponding PR 
[Plane: Fix level-off and throttle without airspeed sensor](https://github.com/ArduPilot/ardupilot/pull/28776)

**Comment by rmackay9** [comment]
Hi @menschel, thanks, added to the list!

**Comment by MattKear** [comment]
There is a missing battery monitor option in the param meta data that will need to be brough across.  PR here:
#29017

**Comment by timtuxworth** [comment]
This: Holybro M10 waiting to configure GPS1 ([discussion](https://discuss.ardupilot.org/t/fc-tries-to-config-gps-constantly-when-trying-to-arm-in-loiter-mode/129930/7), https://github.com/ArduPilot/ardupilot/pull/29320) -- resolved for -beta5

This fixed a problem I was seeing with the QioTek M10S GPS on master.

**Comment by mikedkelly** [comment]
When I arm my plane the status changes to DCM Active and it throws continual Bad AHRS errors. As soon as I disarm it changes back to EKF3 Active.

**Comment by Ryanf55** [comment]
> When I arm my plane the status changes to DCM Active and it throws continual Bad AHRS errors. As soon as I disarm it changes back to EKF3 Active.

Can you set LOG_DISARMED to 1 and get logs of it? Have you tried to reproduce on 4.5?

**Comment by mikedkelly** [comment]
Here is the log and my params. I confirmed this happens on 4.5 and 4.6 beta 4. I guess I didn't notice it before.
[36 3-10-2025 10-31-44 AM.zip](https://github.com/user-attachments/files/19166847/36.3-10-2025.10-31-44.AM.zip)

**Comment by rmackay9** [comment]
Hi @mikedkelly, @Ryanf55,

It's tough to do support on this issue so I've created a new discussion here https://discuss.ardupilot.org/t/mikedkellys-report-on-dcm-active-and-bad-ahrs/131592

**Comment by timtuxworth** [comment]
There is definitely a problem with the M10S @tridge . I had a plane that was struggling to get satellites on it's M10S, so I replaced it with a brand new one out of the box. After 2 hours after booting with master no (zero satellites), I flashed 4.5.7 and it immediately came up with 11 satellites.

**Comment by andyp1per** [comment]
> There is definitely a problem with the M10S [@tridge](https://github.com/tridge) . I had a plane that was struggling to get satellites on it's M10S, so I replaced it with a brand new one out of the box. After 2 hours after booting with master no (zero satellites), I flashed 4.5.7 and it immediately came up with 11 satellites.

I tend to agree - seeing issues on an M10 on my Pavo20.

**Comment by timtuxworth** [comment]
> I tend to agree - seeing issues on an M10 on my Pavo20.

Try building the latest master Andy (more recent than 23rd March) - do you still see the problem?

**Comment by rmackay9** [comment]
Hi @andyp1per @timtuxworth,

This issue probably isn't the best place to discuss this.  Can we move the discussion to [this new topic](https://discuss.ardupilot.org/t/timts-dronecan-ublox-m10-num-sats-issue/132353)?

**Comment by ilia-enginer** [comment]
hello. I have a problem with using ardupilot sub of any version. after flashing usb firmware, the usb connection with the mission planner is lost. this is a bug in all stable versions of sub for speedebeef405v3. only the latest firmware version for developers using the speedebeef405v3 flight controller works properly

**Comment by peterbarker** [comment]
Converting board `hwdef.dat` from 4.7 (and later) to 4.6:
```
Rename AP_COMPASS_PROBING_ENABLED to HAL_PROBE_EXTERNAL_I2C_COMPASSES (see https://github.com/ArduPilot/ardupilot/pull/30317)
```

**Comment by ThuanDuy2K** [comment]
Hi,
I have this crash when takeoff but i exactly don't know what happened
- Hardware is ok.
- PreArm check is 'ready to arm'
https://discuss.ardupilot.org/t/crash-arducopter-4-6-2-desroll-and-despitch-error-when-arming/139726/6

**Comment by DavideLentini** [comment]
Hello everybody, I'm experiencing this problem with 4.6.3 , to just not repeat my self I'll just post the link of Ardupilot discuss post , thanks 

https://discuss.ardupilot.org/t/unexpected-yaw-behavior-and-the-ignore-mavlink-overrides-parameter-not-working/143722
